src/main/java/redis/clients/jedis/JedisClusterInfoCache.java

```diff
@@ -38,6 +38,8 @@ public class JedisClusterInfoCache {
   private final Map<String, ConnectionPool> nodes = new HashMap<>();
   private final ConnectionPool[] slots = new ConnectionPool[Protocol.CLUSTER_HASHSLOTS];
   private final HostAndPort[] slotNodes = new HostAndPort[Protocol.CLUSTER_HASHSLOTS];
+  private List<ConnectionPool>[] replicaSlots;
+  private List<HostAndPort>[] replicaSlotNodes;
```

Comment:
We can use `final` just by adding an `else` block in constructor.
```suggestion
  private final List<ConnectionPool>[] replicaSlots;
  private final List<HostAndPort>[] replicaSlotNodes;
```